### PR TITLE
add info log to log tunnel time for local tunnel

### PR DESCRIPTION
**Pull request by windtalker**
### What problem does this PR solve?

Issue Number: ref #6233

Problem Summary:

### What is changed and how it works?
For both sync tunnel and async tunnel, when a tunnel is finished, there is a info log to record the tunnel cost time, but for local tunnel, there is no such logs, this pr add this log for local tunnel.
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by yibin87 — APPROVED** [review]
LGTM

File: dbms/src/Flash/Mpp/ExchangeReceiver.cpp [hunk 1]
```diff
@@ -675,20 +675,23 @@ void ExchangeReceiverBase<RPCContext>::readLoop(const Request & req)
     });
 
     CPUAffinityManager::getInstance().bindSelfQueryThread();
+    Stopwatch watch;
     bool meet_error = false;
     String local_err_msg;
     String req_info = fmt::format("tunnel{}+{}", req.send_task_id, req.recv_task_id);
+    ReceiverMode recv_mode = req.is_local ? ReceiverMode::Local : ReceiverMode::Sync;
+    UInt64 waiting_task_time = 0;
```
Comment (xzhangxian1008) [inline dbms/src/Flash/Mpp/ExchangeReceiver.cpp:683]: Maybe `sending_data_time` is better?

File: dbms/src/Flash/FlashService.cpp [hunk 2]
```diff
@@ -422,6 +422,7 @@ grpc::Status FlashService::EstablishMPPConnection(grpc::ServerContext * grpc_con
     auto task_manager = tmt_context.getMPPTaskManager();
     std::chrono::seconds timeout(10);
     auto [tunnel, err_msg] = task_manager->findTunnelWithTimeout(request, timeout);
+    auto waiting_task_time = watch.elapsedMilliseconds();
```
Comment (xzhangxian1008) [inline dbms/src/Flash/FlashService.cpp:425]: Maybe `sending_data_time` is better?

**Review by xzhangxian1008 — APPROVED** [review]
Other LGTM

File: dbms/src/Flash/Mpp/ExchangeReceiver.cpp (hunk 1 above)
Comment (windtalker) [reply dbms/src/Flash/Mpp/ExchangeReceiver.cpp:683]: It is the time that `findTunnelWithTimeout` takes, so I think `waiting_task_time` is better here.

**Pull request merged**